quic/s2n-quic-qns/src/tls.rs

```diff
@@ -212,6 +225,41 @@ pub mod s2n_tls {
             s2n_quic_core::crypto::tls::testing::certificates::KEY_PEM.into_private_key()?
         })
     }
+
+    #[derive(Clone)]
+    pub struct SessionTicketHandler {
+        ticket_storage: Arc<Mutex<VecDeque<Vec<u8>>>>,
+    }
+
+    impl Default for SessionTicketHandler {
+        fn default() -> SessionTicketHandler {
+            SessionTicketHandler {
+                // Cap session ticket storage at 10 tickets
+                ticket_storage: Arc::new(Mutex::new(VecDeque::with_capacity(10))),
```

Comment:
this won't actually cap it; it'll just pre-allocate enough for 10 entries (which we should probably not do, since we don't always get a ticket).